<!---->
<launch>
<!-- apply laser filter for front scan-->
<node pkg="laser_filters" type="scan_to_cloud_filter_chain" output="screen" name="cloud_filter_front">
<remap from="scan" to="/lms1xx_front/scan" />
<remap from="cloud_filtered" to="front_cloud_filtered"/>
<param name="high_fidelity" value="true" />
<param name="target_frame" type="string" value="base_link" />
<rosparam command="load" file="$(find fieldrobot_05)/launch/filters/filter_one.yaml" />
</node>

<!-- apply laser filter for back scan -->
<node pkg="laser_filters" type="scan_to_cloud_filter_chain" output="screen" name="cloud_filter_back">
<remap from="scan" to="/lms1xx_back/scan" />
<remap from="cloud_filtered" to="back_cloud_filtered"/>
<param name="high_fidelity" value="true" />
<param name="target_frame" type="string" value="base_link" />
<rosparam command="load" file="$(find fieldrobot_05)/launch/filters/filter_two.yaml" />
</node>


</launch>
